modules/manage/partials/remote-read-replicas.adoc

```diff
@@ -36,11 +34,9 @@ endif::[]
 
 You need the following:
 
-* An origin cluster with xref:{tiered-storage-link}#set-up-tiered-storage[Tiered Storage] set up.
+* An origin cluster with xref:{tiered-storage-link}#set-up-tiered-storage[Tiered Storage] set up. Note that multi-region buckets or containers are not supported.
```

Comment:
My understanding is that it's specifically the object storage bucket (for the TS setup) that has to be single-region and not multi-region. Would like to double check that this is the best place in this doc to add a heads-up regarding multi-region?

Reply:
I'm OK with keeping it this way. RRR will work with both single reagion and multi-region buckets but we only test with single region.